### ISSUE TITLE
BannerActionText when `isSticky` is true is still associated to an `aria-labelledby`

### ISSUE THREAD

**Issue by mannycarrera4** [Workday/canvas-kit#2317]
## 🐛 Bug Report

When `isSticky` is true for the Banner component, al though the `BannerActionText` is still visible, there's an `aria-labelledby` associated with the hidden text.

## To Reproduce

https://workday.github.io/canvas-kit/?path=/story/components-indicators-banner--sticky

## Expected Behavior

There's shouldn't be an `aria-labelledby` when `isSticky` true and the BannerActionText should have `accessibleHide` applied to it.

**Comment by mannycarrera4** [comment]
Closing, this was fixed in https://github.com/Workday/canvas-kit/pull/2318
